File: gameplay/js/model/HexLocation.js
var catan = catan || {};
catan.models = catan.models || {};

catan.models.HexLocation = (function HexLocationNameSpace () {

	var HexLocation = (function () {

		/**
		HexLocation Class 
		<pre>
		</pre>
		@class HexLocation 
		@constructor
		@param {int} x
		@param {int} y
		*/
		function HexLocation(x, y) {
		    this.x = x;
			this.y = y;
		}

		/**
		GetX method
		<pre>
		</pre>
		@method getX
		@return {int} x
		*/
		HexLocation.prototype.getX = function() {
		    return this.x;
		}

		/**
		GetY method
		<pre>
		</pre>
		@method getY
		@return {int} y
		*/
		HexLocation.prototype.getY = function() {
		    return this.y;
		}

		return HexLocation;
	})();


	return HexLocation;
})();

File: server/src/catan/model/HexLocation.java
package catan.model;

public class HexLocation {

	private String x;
	private String y;
	
	public HexLocation () {}
	
	public HexLocation(String x, String y) {
		
		this.x = x;
		this.y = y;
	}
	
	public HexLocation(int x, int y) {
		
		this.x = "" + x;
		this.y = "" + y;
	}
	
	public String getX() {
		return x;
	}
	public void setX(String x) {
		this.x = x;
	}
	public String getY() {
		return y;
	}
	public void setY(String y) {
		this.y = y;
	}

	@Override
	public String toString() {
		return "HexLocation [x=" + x + ", y=" + y + "]";
	}

}
